config/jobs/build-drivers/build-new-bottlerocket.yaml

```diff
@@ -6,7 +6,7 @@ presubmits:
     agent: kubernetes
     branches:
       - ^master$
-    run_if_changed: '^driverkit/config/[a-z0-9.+-]{5,}/(.+/)?bottlerocket*'
+    run_if_changed: '^driverkit/config/[a-z0-9.+-]{5,}/(.+/)?bottlerocket.+'
```

Comment:
Also, can we use `_.*` everywhere? We always expect an `_` after the target name.

Reply:
Done myself :)

Reply:
Thank you @FedeDP